src/10_functions.py

```diff
@@ -1,12 +1,20 @@
 # Write a function is_even that will return true if the passed-in number is even.
 
-# YOUR CODE HERE
+
+def is_even(x):
+    if x % 2 == 0:
+        return True
+    else:
+        return False
```

Comment:
Awesome work. Here we can also write `return x % 2 == 0` which will return a Boolean itself